fvm/bootstrap.go

```diff
@@ -678,8 +678,8 @@ transaction(clusterWeights: [{String: UInt64}]) {
 		randomSource: %s,
 		collectorClusters: clusters,
         // NOTE: clusterQCs and dkgPubKeys are empty because these initial values are not used
-		clusterQCs: [],
-		dkgPubKeys: [],
+		clusterQCs: [] as [FlowClusterQC.ClusterQC],
+		dkgKeys: [] as [String],
```

Comment:
Did type checking fail when just using `[]`?

Reply:
Yes. I had to do a lot of similar changes to core contracts as well.